tee_supplicant/src/tee_supplicant.c

```diff
@@ -340,9 +346,20 @@ static int tee_supp_init(const struct device *dev)
 		LOG_ERR("Only shared memory registration supported");
 		return -EINVAL;
 	}
+	arg = k_malloc(sizeof(struct thread_arg));
+	if (!arg) {
+		LOG_ERR("Unable to allocate thread arg");
+		return -ENOMEM;
+	}
+	arg->tee_fs_root = CONFIG_OPTEE_STORAGE_ROOT;
+
+	rc = mkpath(CONFIG_OPTEE_STORAGE_ROOT);
```

Comment:
You are using `mkpath` here but introduce it in the next patch only

Reply:
Fixed